### PR TITLE
Javascript code execution

### PR DESCRIPTION
**Pull request by kliver98**
Execution:
image

### CODE REVIEW

File: test/i-frame.spec.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { browser } from 'protractor';
+import { IFramePage } from '../src/page';
+
+describe('Given a practice IFrame page', () => {
```
Comment (holgiosalos) [inline test/i-frame.spec.ts]: what about? 
`Given an IFrame page`

File: test/i-frame.spec.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  beforeAll(async () => {
+    browser.waitForAngularEnabled(false);
+    await browser.get('https://demoqa.com/frames');
+  });
+
+  describe('When the height of IFrame wants to be modify', () => {
```
Comment (holgiosalos) [inline test/i-frame.spec.ts]: `When the height is modified`

Comment (kliver98) [reply test/i-frame.spec.ts]: Done

File: test/i-frame.spec.ts (hunk 1 above)
Comment (kliver98) [reply test/i-frame.spec.ts]: Done

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**